packages/types-algosdk/types/index.d.ts

```diff
@@ -195,10 +195,25 @@ declare module 'algosdk' {
     v: number;
   }
 
-  export interface MultiSigAccount {
+  /**
+   * Src: https://github.com/algorand/js-algorand-sdk/blob/develop/src/types/multisig.ts#L6
+   */
+  // TODO: use latest typings from SDK
+  export interface MultisigMetadata {
+    /**
+     * Multisig version
+     */
     version: number;
+
+    /**
+     * Multisig threshold value. Authorization requires a subset of signatures,
+     * equal to or greater than the threshold value.
+     */
     threshold: number;
-    // array of base32 encoded addresses
+
+    /**
+     * A list of Algorand addresses representing possible signers for this multisig. Order is important.
```

Comment:
here you have mentioned order is important but above in threshold, it is mentioned we require a subset of signatures, 
why order here is important ?

Reply:
[https://developer.algorand.org/docs/features/accounts/create/#multisignature](https://developer.algorand.org/docs/features/accounts/create/#multisignature)